Reading D:/Software/Modelsim/tcl/vsim/pref.tcl

# 10.6d

# do sim_build/runsim.do
# Model Technology ModelSim SE-64 vmap 10.6d Lib Mapping Utility 2018.02 Feb 24 2018
# vmap -c 
# ** Warning: vmap will not overwrite local modelsim.ini.
# Model Technology ModelSim SE-64 vmap 10.6d Lib Mapping Utility 2018.02 Feb 24 2018
# vmap work sim_build/work 
# Modifying modelsim.ini
# Model Technology ModelSim SE-64 vlog 10.6d Compiler 2018.02 Feb 24 2018
# Start time: 20:58:44 on Jan 22,2022
# vlog -work work "+define+COCOTB_SIM" -sv -timescale 1ns/1ps -mfcu "+acc" D:/Gitlib/test/cocotb/src/test_top.sv 
# -- Compiling module test_top
# 
# Top level modules:
# 	test_top
# End time: 20:58:44 on Jan 22,2022, Elapsed time: 0:00:00
# Errors: 0, Warnings: 0
# ** Warning: (vsim-159) Mode option -64 is not supported in this context and will be ignored.
# vsim -onfinish exit -pli "D:/software/miniconda/lib/site-packages/cocotb/libs/cocotbvpi_modelsim.dll" sim_build/work.test_top 
# Start time: 20:58:44 on Jan 22,2022
# ** Note: (vsim-3812) Design is being optimized...
# ** Note: (vsim-3865) Due to PLI being present, full design access is being specified.
# //  ModelSim SE-64 10.6d Feb 24 2018
# //
# //  Copyright 1991-2018 Mentor Graphics Corporation
# //  All Rights Reserved.
# //
# //  ModelSim SE-64 and its associated documentation contain trade
# //  secrets and commercial or financial information that are the property of
# //  Mentor Graphics Corporation and are privileged, confidential,
# //  and exempt from disclosure under the Freedom of Information Act,
# //  5 U.S.C. Section 552. Furthermore, this information
# //  is prohibited from disclosure under the Trade Secrets Act,
# //  18 U.S.C. Section 1905.
# //
# Loading sv_std.std
# Loading work.test_top(fast)
# Loading D:/software/miniconda/lib/site-packages/cocotb/libs/cocotbvpi_modelsim.dll
#      -.--ns INFO     cocotb.gpi                         ..mbed\gpi_embed.cpp:78   in set_program_name_in_venv        Did not detect Python virtual environment. Using system-wide Python interpreter
# 
#      -.--ns INFO     cocotb.gpi                         ..\gpi\GpiCommon.cpp:99   in gpi_print_registered_impl       VPI registered
D:\software\miniconda\lib\site-packages\cocotb\scheduler.py:541: FutureWarning: "Unwatched" tasks that throw exceptions will not cause the test to fail. See issue #2664 for more details.
  warnings.warn(
#      0.00ns INFO     Running on ModelSim SE-64 version 10.6d 2018.02
# 
#      0.00ns INFO     Running tests with cocotb v1.6.1 from D:\software\miniconda\lib\site-packages\cocotb
# 
#      0.00ns INFO     Seeding Python random module with 1642856326
# 
#      0.00ns WARNING  Pytest not found, assertion rewriting will not occur
# 
#      0.00ns INFO     Found test tb_test_top.run_test
# 
#      0.00ns INFO     running run_test (1/1)
# 
#      0.00ns INFO     >>>>>>>>>> user: create input_driver
# 
#      0.00ns INFO     >>>>>>>>>> user: create output_monitor
# 
#      0.00ns INFO     >>>>>>>>>> user: create clock_domain
# 
#      0.00ns INFO     >>>>>>>>>> user: tb_adder create
# 
#      0.00ns INFO     >>>>>>>>>> user: start din_drv thread
# 
#      0.00ns INFO     >>>>>>>>>> user: start dout_mon thread
# 
#      0.00ns INFO     >>>>>>>>>> user: clock_rst start
# 
#      0.00ns INFO     >>>>>>>>>> user: tb_start
# 
#      0.00ns INFO     >>>>>>>>>> user: add trans 3 + 5
# 
# (3, 5)
# 
#      0.00ns INFO     >>>>>>>>>> user: pop din start
# 
#      0.00ns INFO     >>>>>>>>>> user: get a transaction in input driver
# 
# z
# 
# zzzzzzzz
# 
# zzzzzzzz
# 
# x
# 
#      0.00ns ERROR    Exception raised by this forked coroutine
# 
#      0.00ns INFO     run_test failed
# 
#                      Traceback (most recent call last):
# 
#                        File "D:\Gitlib\test\cocotb\tb_test_top.py", line 92, in task_monitor
# 
#                          if self.dout_vld.value.integer:
# 
#                        File "D:\software\miniconda\lib\site-packages\cocotb\binary.py", line 336, in integer
# 
#                          return self._convert_from(self._str)
# 
#                        File "D:\software\miniconda\lib\site-packages\cocotb\binary.py", line 231, in _convert_from_unsigned
# 
#                          return int(x.translate(_resolve_table), 2)
# 
#                        File "D:\software\miniconda\lib\site-packages\cocotb\binary.py", line 77, in __missing__
# 
#                          return self.resolve_x(key)
# 
#                        File "D:\software\miniconda\lib\site-packages\cocotb\binary.py", line 59, in resolve_error
# 
#                          raise ValueError("Unresolvable bit in binary string: '{}'".format(chr(key)))
# 
#                      ValueError: Unresolvable bit in binary string: 'x'
# 
#      0.00ns INFO     **************************************************************************************
# 
#                      ** TEST                          STATUS  SIM TIME (ns)  REAL TIME (s)  RATIO (ns/s) **
# 
#                      **************************************************************************************
# 
#                      ** tb_test_top.run_test           FAIL           0.00           0.00          0.50  **
# 
#                      **************************************************************************************
# 
#                      ** TESTS=1 PASS=0 FAIL=1 SKIP=0                  0.00           0.02          0.06  **
# 
#                      **************************************************************************************
# 
#                      
# 
# ** Note: $finish
#    Time: 1 ps  Iteration: 0  Instance: /test_top
# End time: 20:58:46 on Jan 22,2022, Elapsed time: 0:00:02
# Errors: 0, Warnings: 1
